pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -140,7 +140,18 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 		}
 	}
 
-	if len(availableMembers) > len(etcdMembers)/2 {
+	if len(availableMembers) == len(etcdMembers) {
```

Comment:
what is the status output when 1 member (or minority members) is unhealthy?

Reply:
See the else part (L154-165)

Reply:
I am not sure I understand the question, Alay. Do you want a different status when minority members are unhealthy?

Reply:
no, I missed the `else if` was just trying to wrap my head around all the cases